lmnet/executor/train.py

```diff
@@ -112,7 +99,7 @@ def start_training(config):
         else:
             loss = model.loss(output, labels_placeholder)
         opt = model.optimizer(global_step)
```

Comment:
Although I've never tried that. I heard that distributed training had a larger batch size. So the gradient is more accurate. But the update frequency is lower. To get a better performance, we usually need to enlarge learning rate to get a better performance. It should be something like `opt = tf.train.MomentumOptimizer(0.01 * hvd.size())`.

Reply:
I has some experiments about that. The technique you mentioned is quite effective.

It would be a good idea to recommend to increase the learning rate in proportion to the batch size. However, automatically changing the learning rate which is set by a user, might confuse the user.